### PR TITLE
Add enable ipv6  network configuration

### PR DESCRIPTION
**Pull request by mrjana**

### CODE REVIEW

File: pkg/netlabel/labels.go~ [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package netlabel
```
Comment (mavenugo) [inline pkg/netlabel/labels.go~]: Please remove this file.

File: libnetwork_test.go [hunk 2]
```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/docker/docker/pkg/reexec"
```
Comment (mavenugo) [inline libnetwork_test.go]: Can you please add a test for the newly added `enable_ipv6` support ?

File: pkg/netlabel/labels.go~ (hunk 1 above)
Comment (mrjana) [inline pkg/netlabel/labels.go~]: Oops.. didn't know the emacs file got in. Will remove it.

File: libnetwork_test.go (hunk 2 above)
Comment (mrjana) [inline libnetwork_test.go]: Will add.

**Comment by mavenugo** [comment]
Thanks for taking care of the comments. LGTM.

**Pull request merged**